contracts/modules/token/TokenController.sol

```diff
@@ -171,6 +171,10 @@ contract TokenController is ITokenController, LockHandler, MasterAwareV2 {
         continue;
       }
       uint256 amount = locked[users[i]]["CLA"].amount;
+
+      // TODO: test this function
+      assert (amount == 0);
```

Comment:
TODO: remove assert

Reply:
I don't see a method of setting the `locked` storage value. I tried to investigate where it gets set, but it seems none of our tests do. Is this fully implemented?

Reply:
It was implemented, it's functionality that was removed from the contracts. We'll test it in fork tests, skip unit tests for this part.